net-dns/coredns/files/coredns.initd

```diff
@@ -18,3 +18,13 @@ start_stop_daemon_args="--user ${user} --group ${group} \
 depend() {
 	after net
 }
+
+if [[ -n "${COREDNS_HEALTHCHECK_URI}" ]]; then
+        healthcheck_delay=20
+        healthcheck_timer=30
+
+        healthcheck() {
+                command -v wget || return 0
+                wget -Oq- "${COREDNS_HEALTHCHECK_URI}"
+        }
+fi
```

Comment:
Does this really make a difference? Wouldn't you always want it to be `127.0.0.1:8092/health`?

Reply:
It is not enabled by default by the upstream, port can also be different, as well as bind address (for example, I use 127.1.1.1)